docs/extend/legacy_plugins.md

```diff
@@ -42,10 +42,6 @@ Follow the instructions in the plugin's documentation.
 
 The sections below provide an inexhaustive overview of available plugins.
 
-<style>
-#DocumentationText  tr td:first-child { white-space: nowrap;}
-</style>
```

Comment:
Oh, this was a big hack indeed; did we have a replacement for this in the new layout? (preventing the column from wrapping?)